bundle/manifests/cluster-group-upgrades-operator.clusterserviceversion.yaml

```diff
@@ -416,6 +416,8 @@ spec:
                 command:
                 - /manager
                 env:
+                - name: CONTROLLER_IMG
```

Comment:
Kindly move the CONTROLLER_IMG right after INSECURE_CALL_GRAPH (as it was done on in.yaml)